ajantv2/src/ntv2driverinterface.cpp

```diff
@@ -591,7 +591,10 @@ bool CNTV2DriverInterface::DriverGetBitFileInformation (BITFILE_INFO_STRUCT & bi
 	if (!::NTV2DeviceHasSPIFlash(_boardID))
 		return false;
 
-	ParseFlashHeader(bitFileInfo);
+	if (!ParseFlashHeader(bitFileInfo)) {
```

Comment:
@janbrothanek Could you remove the braces to match the existing coding style in that function (e.g. as in line 591-592)?

Reply:
@janbrothanek Please open a separate PR for this change.

As @mrbillaja says that the change to "ajaanc/src/ancillarylist.cpp" is not needed I am going to close this PR.